src/test/java/com/epam/ta/reportportal/dao/suite/LaunchRepositoryTest.java

```diff
@@ -225,6 +225,14 @@ public void identifyStatus() {
 		assertTrue(failed);
 	}
 
+	@Test
+	public void hasRetries() {
+
+		final boolean hasRetries = launchRepository.hasRetries(100L);
+		assertTrue(hasRetries);
```

Comment:
should be negative test as well to prove that method can return both of values

Reply:
added